### PR TITLE
redesigned toolbar based upon zeplin

### PR DESCRIPTION
**Pull request by hollydo**
resolves #

addresses #

# Motivation and context

# Screenshots
| before | after |
|---|---|
| Screen Shot 2019-09-17 at 11 20 32 PM | Screen Shot 2019-09-17 at 11 17 02 PM |

# What I did
-

### CODE REVIEW

**Comment by emillipede** [comment]
thank you @hollydo!! the toolbar looks so much better 

a couple of questions and issues I noticed: 

- in the zeplin mockup, the button widths vary with the length of the words. right now, some of the buttons look a little squished, and "species" and "family" have extra room.
Screen Shot 2019-09-18 at 12 38 04 PM
Screen Shot 2019-09-18 at 12 38 15 PM

- the dropdown menu for species gets hidden behind the white border:
Screen Shot 2019-09-18 at 12 31 33 PM do you know how to prevent that issue? I think @theswerd encountered this problem a while ago, too. Ben, did you find any ways to resolve it?

- the toolbar looks anchored to the top corner. in the mockup, it floats over the map with some map visible in the top left corner.

- the size of the text in the address and species search bars looks larger than in the zeplin mockup. some of the content doesn't fit. the "zoom to" / "group by" / "find species" labels in zeplin are a slightly different color, too (#444444)

- since the side panel will become substantially narrower, I think it makes sense to extend the toolbar (into some of that leftover space towards the right)

- what's the benefit of the hand/cursor switching when it moves over the "group by" buttons?

please let me know what you think. thank you again!

File: public/index.html [hunk 1]
```diff
@@ -56,20 +56,26 @@
       <!-- Toolbar -->
       <div id="toolbar">
 
+        
+        <div class="address-container">
+          <label class="toolbar-label">ZOOM TO ADDRESS</label>
```
Comment (emillipede) [inline public/index.html]: ZOOM TO LOCATION

Comment (emillipede) [reply public/index.html]: sorry, the all caps in the comment looks very intense

**Comment by hollydo** [comment]
> * what's the benefit of the hand/cursor switching when it moves over the "group by" buttons?

The hand/cursor on hover for the "group by" buttons, signifies to the user that these are buttons and can be clicked.

**Comment by emillipede** [comment]
thank you so much @hollydo ! 

I added a couple of small formatting changes to the toolbar to try match the zeplin. please let me know what you think. 

How do you change the text size for the address and select2 bars? I think it makes sense for the button text and the text inside the search bars to be sized the same - in part to match the mockup and just because it'll be easier to read - but I haven't figured out how to do it (same for the sidebar width, but that's a separate issue).

I know the cursor changes a little bit on the current site, but styling the cursor is something we've hadn't really talked about - it's a good idea, thanks for adding it! there are so many cursor options I hadn't considered

**Comment by emillipede** [comment]
in order to get feedback from UFTF, merging WIP

**Pull request merged**